docs/tutorial/part-one/index.md

```diff
@@ -112,27 +112,29 @@ Change the color to "pink". Then to "tomato".
 
 2.  Add some paragraph text.
 
-```jsx{5-6}
+```jsx{3,5-6,8}
```

Comment:
Is highlighting 3rd and 8th line intended?

Reply:
Lines 3 and 8 introduce opening and closing brackets.

```jsx
import React from "react"

export default () => <div style={{ color: `blue` }}>Hello Gatsby!</div>
```

to

```jsx
import React from "react"

export default () => (
  <div style={{ color: `tomato` }}>
    <h1>Hello Gatsby!</h1>
    <p>What a world.</p>
  </div>
)
```

Do you think the highlights aren’t necessary?

Reply:
Not at all. I was just checking with You if this wasn't something that slipped through unnoticed.